tests/org.codechecker.eclipse.rcp.it.tests/src/org/codechecker/eclipse/plugin/PluginTest.java

```diff
@@ -3,30 +3,21 @@
 import java.io.File;
```

Comment:
This existing test is modified to take advantage of the new utilites like file loading, root test suite.